Shared/Samples/Add raster from service/AddRasterFromServiceView.swift

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddRasterFromServiceView: View {
+    /// The error shown in the error alert.
+       @State private var error: Error?
```

Comment:
The whole file's indentation is off. Please select all then in Xcode > Editor > Structure > Reindent to fix it.